src/App.js

```diff
@@ -1,11 +1,24 @@
-
+import React from 'react';
+import { BrowserRouter, Route } from 'react-router-dom';
+import FetchedPosts from './components/FetchedPosts'
+import Card from './components/Card'
 
 function App() {
+
 	return (
 		<div>
+			<BrowserRouter>
+				<h2>Super Heroes</h2>
```

Comment:
Please create file where you'll store all string literals (constants.js for example).
Better to create an object.

Reply:
Didn't implemented